src/devices/Si7021/samples/Program.cs

```diff
@@ -19,8 +21,17 @@ static void Main(string[] args)
             {
                 while (true)
                 {
-                    Console.WriteLine($"Temperature: {sensor.Temperature.Celsius}℃");
-                    Console.WriteLine($"Humidity: {sensor.Humidity}%");
+                    var tempValue = sensor.Temperature;
+                    var humValue = sensor.Humidity;
+                    
+                    Console.WriteLine($"Temperature: {tempValue.Celsius:0.#}\u00B0C");                    
+                    Console.WriteLine($"Relative humidity: {humValue:0.#}%");
+                    Console.WriteLine($"Heat index: {WeatherHelper.CalculateHeatIndex(tempValue, humValue).Celsius:0.#}\u00B0C");
```

Comment:
Pick only 1 or 2 parameter per sample (for temperature and pressure sensors few more are fine but for the rest let's focus on showing the device and not everything)

Reply:
Done (pending commit)